frontend/src/components/templates/UserEnrollmentList/Enrollment.js

```diff
@@ -34,6 +36,27 @@ const Enrollment = ({
       .value();
   }, [events]);
 
+  const getNextStepButton = () => {
```

Comment:
this can be a sub component that uses useDataProvider or useFullDataProvider hooks